debian/build_test.go

```diff
@@ -94,24 +59,25 @@ func TestGetKubeadmDependencies(t *testing.T) {
 		deps    []string
 	}{
 		{
-			name:    "simple test",
-			version: "1.10.0",
+			name:    "minimum supported kubernetes",
+			version: "1.13.0",
 			deps: []string{
-				"kubelet (>= 1.6.0)",
-				"kubectl (>= 1.6.0)",
+				"kubelet (>= 1.12.0)",
+				"kubectl (>= 1.12.0)",
 				"kubernetes-cni (>= 0.7.5)",
 				"${misc:Depends}",
+				"cri-tools (>= 1.13.0)",
```

Comment:
My impression is that cri-tools is intended to interact with kubelet, so cri-tools being at 1.13.0 when kubelet is at 1.12.0 seems off?

Reply:
@Random-Liu as the author of https://github.com/kubernetes-sigs/cri-tools/issues/297 do you have any insight here?

Reply:
I might be wrong but cri-tools interact with the CRI, pretty sure there's no direct kubelet interaction. crictl talks to the CRI endpoint for containerd etc.

I think the confusion here is "kubelet CRI" in the docs which I'm pretty sure is just referring to the CRI API, given that the spec used to only exist as a package in kubelet.

Generally CRI has been fairly compatible recently so this should probably be fine.

These specs not being versioned per kubernetes release branch is a disaster though, independent of this PR.

Reply:
@Random-Liu is OOO for a few weeks so I wouldn't expect a response here. (Otherwise be more OOO! :upside_down_face:)

Reply:
The version matrix is here: https://github.com/kubernetes-sigs/cri-tools. And yep, cri-tools v1.13 is comptable with kubelet v1.12, because CRI didn't change during those two versions.

Reply:
>because CRI didn't change during those two versions.

I was pretty sure this was the case, but where can I look to find this out?  The table in cri-tools' repo doesn't make this clear